charts/trento-server/values.yaml

```diff
@@ -17,6 +17,9 @@ global:
   rabbitmq:
     name: rabbitmq
     servicePort: 5672
+  certManager:
+    enabled: false
+    clusterIssuer: "letsencrypt-production"
```

Comment:
This can be removed: we don't want to couple the server chart with cert manager.